virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,73 @@
+* Install the xref:../virt/install/installing-virt-web.adoc#installing-virt-web[Install the {VirtProductName} Operator] and the xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[`virtctl` tool].
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Red Hat Enterprise Linux virtual machines] using the wizard.
+* Create a Microsoft Windows virtual machine
+** xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Create and upload a Microsoft Windows boot source].
+** Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Microsoft Windows virtual machines] using the wizard.
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a virtual machine using xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-to-the-terminal_virt-accessing-vm-consoles[the terminal] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[the serial console] in the {product-title} web console.
+* Connect to a virtual machine using SSH:
+** xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[Create a service] to expose a route for the virtual machine.
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+
+Manage a virtual machine.
+
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Network options:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[Configure a Linux bridge network], in addition to the default pod network.
+* Install the xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[SR-IOV Operator], configure a xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[SR-IOV network device], and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[attach a virtual machine to the SR-IOV network].
+
+Microsoft Windows virtual machine options:
```

Comment:
Now that I see this in context, I'm not sure whether it's a good idea for us to get this granular. It's not like we offer a whole bunch of other VM options, like PXE booting or attaching a disk. I'm thinking that it might be better not to include the Windows VM options.

Reply:
OK, I'll remove the Windows options section. I saved a copy of all the original links in a text file if we need to re-add them.